File: YSDB/MSDB/Models/daily_report_detail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MSDB.Models
{
    public class daily_report_detail
    {
        [Key]
        public int id { get; set; }
        public int userId { get; set; }
        ///<sumamry>
        /// 工作日期
        ///<sumamry>
        public DateTime workDate { get; set; }

        public DateTime createTime { get; set; }
        public DateTime updateTime { get; set; }
        public int done { get; set; }
        public int commit { get; set; }
        public int reject { get; set; }
        public int moduleId { get; set; }
        public int companyId { get; set; }

    }
}

File: MS/src/main/java/com/yskc/ms/entity/ms/DailyReportDetail.java
package com.yskc.ms.entity.ms;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;

import java.util.Date;

/**
 * @BelongsProject: ms
 * @BelongsPackage: com.yskc.ms.entity.ms
 * @Author: zhangdingfu
 * @CreateTime: 2021-08-03 16:24
 * @Description: 用户审核明细
 */
@TableName("daily_report_detail")
public class DailyReportDetail {
    @TableId
    private Integer id;
    private Integer userId;
    private Date workDate;
    private Date createTime;
    private Date updateTime;
    private Integer done;
    private Integer commit;
    private Integer reject;
    private Integer moduleId;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public Date getWorkDate() {
        return workDate;
    }

    public void setWorkDate(Date workDate) {
        this.workDate = workDate;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Date getUpdateTime() {
        return updateTime;
    }

    public void setUpdateTime(Date updateTime) {
        this.updateTime = updateTime;
    }

    public Integer getDone() {
        return done;
    }

    public void setDone(Integer done) {
        this.done = done;
    }

    public Integer getCommit() {
        return commit;
    }

    public void setCommit(Integer commit) {
        this.commit = commit;
    }

    public Integer getReject() {
        return reject;
    }

    public void setReject(Integer reject) {
        this.reject = reject;
    }

    public Integer getModuleId() {
        return moduleId;
    }

    public void setModuleId(Integer moduleId) {
        this.moduleId = moduleId;
    }
}
